package.json

```diff
@@ -28,9 +28,12 @@
     "update:swap": "node compileBaseSwapOptions.js"
   },
   "dependencies": {
+    "@salad/client": "file:../salad_11.15/client",
```

Comment:
@lauraweindorf  You have to install the salad/client locally for now and mind the directory, change it as needed.

Reply:
ah ok